_posts/2022-01-21-dashboards-plugins-intro.markdown

```diff
@@ -0,0 +1,220 @@
+---
+layout: post
+title: "Introduction to OpenSearch Dashboard Plugins"
+authors:
+  - ashwinpc
+date: 2022-01-21
+categories:
+  - technical-post
+twittercard:
+  description: "Plugins are fundamental to how OpenSearch works, and that similarity extends to OpenSearch Dashboards too..."
+---
+
+Plugins are fundamental to how OpenSearch works, and the similarity extends to OpenSearch Dashboards too. Infact almost everything that you see inside OpenSearch Dashboards is built inside a plugin. As a follow up the blog post on how plugins work for OpenSearch, this post will explore how plugins work for OpenSearch Dashboards.
+
+### What is a plugin
+
+To understand what a plugin in OpenSearch Dashboards is, first it's important to understand what makes up OpenSearch Dashboards.
+
+![DashboardsOverview](/assets/media/blog-images/2022-01-21-dashboards-plugin-intro/dashboards_overview.png){:width="100%"}
```

Comment:
`{:width="100%"}` change it to the class `{: .img-fluid}`

Reply:
Check spelling in image [
![Screen Shot 2022-01-14 at 8 36 50 AM](https://user-images.githubusercontent.com/1152927/149542240-8b7c6522-f96a-445b-97d9-da42cb02d434.png)
](url)